### ISSUE TITLE
Staked LP tokens shouldn't be counted as loss in the DeFI LP PnL calculation

### ISSUE THREAD

**Issue by ulope** [rotki/rotki#3396]
## Abstract

Currently LP tokens staked in unsupported DeFi protocols are shown as a total loss in the DeFi LP PnL list even if there is no exit transaction.

## Motivation

It's confusing having to remember which LP positions are actually exited with profit or loss and which are just not there due to staking.

Related: #3282 

## Specification

Similar to some of the online LP IL trackers (e.g. apy.vision, revert.finance) "missing" LP tokens for which no exit transaction has been found should be (maybe optionally) considered to be staked (or otherwise "in use").

**Comment by LefterisJP** [comment]
Hey @ulope. This loss shows **only** if you have moved the LP tokens to another address. So this is actually very related or even a duplicate of https://github.com/rotki/rotki/issues/1667

It's a very hard issue to fix and needs a bit of thinking. As the LP tokens can be moved to another address, sold, swapped, deposited to another protocol and correlating value between all those events is really hard.

**Comment by ulope** [comment]
Hm maybe my lack of DeFi experience is showing here, but doesn't staking LP tokens by definition involve moving them to another address (e.g. the staking contract)?

**Comment by LefterisJP** [comment]
> Hm maybe my lack of DeFi experience is showing here, but doesn't staking LP tokens by definition involve moving them to another address (e.g. the staking contract)?

No you are correct. I missed the title of your post :sweat: 

But yes indeed that does not change the problem I explained. It's not trivial to track ownership of the LP tokens through all the actions I detailed above.

**Comment by ulope** [comment]
> But yes indeed that does not change the problem I explained. It's not trivial to track ownership of the LP tokens through all the actions I detailed above.

Yeah sure, I get that that's a difficult problem to solve (esp. for the general case).
